pkg/minikube/tunnel/route_test.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2018 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tunnel
+
+import (
+	"net"
+	"reflect"
+	"testing"
+)
+
+func TestRoutingTable(t *testing.T) {
+	tcs := []struct {
+		name         string
+		routingTable routingTable
```

Comment:
to avoid the stuttering, maybe just name the member 'table'?

Reply:
done